cves/kernel/CVE-2012-6657.yml

```
@@ -473,5 +468,5 @@ nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
-CVSS:
+nickname: Socket type non check.
+CVSS:
```

Looks like this vulnerability has not yet been provided a 3.1 CVSS score, but instead you could use the 2.0 CVSS from "https://nvd.nist.gov/vuln/detail/CVE-2012-6657"
```suggestion
CVSS: CVSS:2.0AV:L/AC:L/Au:N/C:N/I:N/A:C
```